sdk/formrecognizer/azure-ai-formrecognizer/src/main/java/com/azure/ai/formrecognizer/models/RecognizedForm.java

```diff
@@ -15,6 +15,8 @@
 @Immutable
 public final class RecognizedForm {
 
+    private float formTypeConfidence;
+    private String modelId;
```

Comment:
these two properties has no way to assign value.
Should add a new constructor function that contains these two properties.

Reply:
I am going to give the new techniques posted by Jonathan a try so that we don't expose those setters or public constructors.
They all should be impl details so hidden.

They anyway shouldn't be in API view.

Reply:
That is awesome. finally we have a way.